caddy/setup/browse.go

```diff
@@ -342,6 +343,17 @@ footer {
 							{{end}}
 						</th>
 					</tr>
+					{{if .CanGoUp}}
+						<tr>
+							<td>
+								<a href="..">
+									<span class="goup">Go up</span>
+								</a>
+							</td>
+							<td>&mdash;</td>
+							<td>&mdash;</td>
+						</tr>
```

Comment:
Nit-pic: Un-intent this whole `<tr>...</tr>` block 1 notch so it is consistent with the other ones.